Firestore/Example/Tests/Integration/API/FIRWriteBatchTests.mm

```diff
@@ -348,7 +348,8 @@ - (void)testReasonableMemoryUsageForLotsOfMutations {
   FIRWriteBatch *batch = [mainDoc.firestore batch];
 
   // >= 500 mutations will be rejected, so use 500-1 mutations
-  for (int i = 0; i != 500 - 1; ++i) {
+  const int maxMutations = 500 - 1;
```

Comment:
I was actually off by one on this, it's >500 mutations that are rejected, not >=500.